src/libraries/System.Net.NameResolution/src/System/Net/NameResolutionMetrics.cs

```diff
@@ -12,42 +12,15 @@ internal static class NameResolutionMetrics
     {
         private static readonly Meter s_meter = new("System.Net.NameResolution");
 
-        private static readonly Counter<long> s_lookupsRequestedCounter = s_meter.CreateCounter<long>(
-            name: "dns-lookups-requested",
-            description: "Number of DNS lookups requested.");
+        private static readonly Histogram<double> s_lookupDuration = s_meter.CreateHistogram<double>(
```

Comment:
We should also set the unit parameter to "s" for seconds.

Reply:
Yes